test/common_utils.py

```diff
@@ -406,6 +406,7 @@ def make_bounding_boxes(
     canvas_size=DEFAULT_SIZE,
     *,
     format=tv_tensors.BoundingBoxFormat.XYXY,
+    num_objects=1,
```

Comment:
This was needed by yet another test I'll flag below. LMK if you want me to rewrite that or we can keep the parameter.